docs/customizing_deis/choosing-a-scheduler.rst

```diff
@@ -122,16 +122,50 @@ The Swarm Scheduler is now active. Commands such as ``deis destroy`` or
 To monitor Swarm Scheduler operations, watch the logs of the swarm-manager
 component, or spy on Docker events directly on the swarm-manager machine:
 
+Kubernetes Scheduler
+--------------------
+
+.. important::
+
+    The Kubernetes Scheduler is a technology preview and is not recommended for
+    production use.
```

Comment:
Here would be a good place to say "The Kubernetes preview can only be used on new clusters. There is no migration path from an existing cluster to the Kubernetes technology preview."